integration/tests/handshake/handshake.test.ts

```diff
@@ -0,0 +1,81 @@
+import type { Server, ServerOptions } from 'node:https';
+
+import { expect, test } from '@playwright/test';
+
+import { constants } from '../../constants';
+import { fs } from '../../scripts';
+import { createProxyServer } from '../../scripts/proxyServer';
+import type { FakeUserWithEmail } from '../../testUtils';
+import { createTestUtils, testAgainstRunningApps } from '../../testUtils';
+
+testAgainstRunningApps({ withPattern: ['next.appRouter.sessionsProd1'] })('handshake flow @handshake', ({ app }) => {
+  test.describe.configure({ mode: 'serial' });
+
+  test.describe('with Production instance', () => {
+    // TODO: change host name
```

Comment:
Do we need to address the TODO here? 
Probably a low priority for the time being, but leaving some high-level instructions here for when we get to this:
- create a new `.clerk.app` host domain
- create and configure a new Clerk prod app
- update the local certificates used by the tests to include the new domain in addition to the existing ones
- finally, update the tests